### PR TITLE
Feature components

### PR DESCRIPTION
**Pull request by algerina**
In this Milestone:
- Deleted all the boilerplate from CRA.
- Created a directory called components.
- Created a new Calculator.js file inside the components directory.
- Created a react component for the Calculator and style it.
- Imported calculator and use it in the main component (App.js).

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import '../calculator.css';
+
+export default function Calculator() {
```
Comment (TedLivist) [inline src/components/calculator.js]: - [x] Right now, your Calculator is a function-based component but according to the project requirements, it should be a class-based component. Kindly convert your component accordingly.

File: src/App.js [hunk 2]
```diff
@@ -1,28 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
```
Comment (TedLivist) [inline src/App.js]: - [x] Kindly convert your App.js component to a class-based one

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @algerina,

Good job so far!
✔️ Your design looks very nice
✔️ You have neat code
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
Kindly look at the images below
- [x] According to the first requirement, you should delete the generated `create-react-app` boilerplate (texts, styles, images). Kindly delete the highlighted files in the images.

amel-public
amel-src

- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @algerina , 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**